plugins/org.jkiss.dbeaver.ext.ui.eula/src/org/jkiss/dbeaver/ext/ui/eula/EULAUtils.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2022 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.ext.ui.eula;
+
+import org.jkiss.dbeaver.Log;
+import org.jkiss.dbeaver.utils.SystemVariablesResolver;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class EULAUtils {
+    private static final Log log = Log.getLog(EULAUtils.class);
+
+
+    //TODO change hardcoded eula version to something more flexible
+    private static final String eulaVersion = "1.0";
+
+    //Only works for packaged version of dbeaver, will not find anything inside development environment
+    private static final String EULA_PATH = SystemVariablesResolver.getInstallPath() + File.separator + "licenses" + File.separator + "dbeaver_license.txt";
```

Comment:
This looks dirty and error-prone. Can we specify it in the application descriptor too?